core/modules/widgets/widget.js

```diff
@@ -249,7 +279,12 @@ Widget.prototype.getStateQualifier = function(name) {
 			}
 			node = node.parentWidget;
 		}
-		var value = $tw.utils.hashString(output.join(""));
+		var value;
+		if(generateTransclusionFootprint) {
+			value = $tw.utils.hashString(output.join("")) + "_" + this.generateTransclusionFootprint();
```

Comment:
Does the `<<qualify` macro and the `<$qualify` widget produce the same, or different hashes?